process/smartContract/builtInFunctions/changeOwnerAddress_test.go

```diff
@@ -16,7 +16,7 @@ func TestChangeOwnerAddress_ProcessBuiltinFunction(t *testing.T) {
 
 	coa := changeOwnerAddress{}
 
-	owner := []byte("sender")
+	owner := []byte("send")
```

Comment:
"owner" ?

Reply:
the length is verified for the validity of the input, so length must be equal.